rules/linux/macos_schedule_task_job_cron.yml

```diff
@@ -0,0 +1,26 @@
+title: Scheduled Cron Task/Job
+id: 7c3b43d8-d794-47d2-800a-d277715aa460
+status: experimental
+description: Detects abuse of the cron utility to perform task scheduling for initial or recurring execution of malicious code. Detection will focus on crontab jobs uploaded from the tmp folder.
+author: Alejandro Ortuno, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1053.003/T1053.003.md
+logsource:
+  category: process_creation
+  product: macos
+detection:
+  selection:
+    ProcessName|endswith:
+      - 'crontab'
```

Comment:
I think that we could bypass the test adding a slash to the processname, this way: `'/crontab'`

Reply:
Regardless of the test, your rules look perfect. Thank you very much!
I will approve your PR and summon maintainers as soon as you will try to bypass the test, regardless of the result (: 
I belive that we just need to give it a try

Reply:
just added the slash @yugoslavskiy but seems that the testing phase is having a different issue now:

>  TypeError: 'NoneType' object is not iterable
Makefile:122: recipe for target 'test-sigma2attack' failed
make: *** [test-sigma2attack] Error 1
Error: Process completed with exit code 2.

Reply:
It seems that it's just broken. Thank you for giving it a try! And for your contribution (: Well done!